flink/src/test/java/org/apache/iceberg/flink/TestFlinkFilters.java

```diff
@@ -0,0 +1,361 @@
+    Optional<org.apache.iceberg.expressions.Expression> actual = FlinkFilters.convert(expr);
+    Assert.assertTrue(actual.isPresent());
+    And and = (And) actual.get();
+    And expected = (And) org.apache.iceberg.expressions.Expressions.and(
+        org.apache.iceberg.expressions.Expressions.equal("field1", 1),
+        org.apache.iceberg.expressions.Expressions.equal("field2", 2L));
+
+    Assert.assertEquals(expected.op(), and.op());
+    Assert.assertEquals(expected.left().op(), and.left().op());
+    Assert.assertEquals(expected.right().op(), and.right().op());
+  }
+
+  @Test
+  public void testOr() {
+    Expression expr = resolve(
+        Expressions.$("field1").isEqual(Expressions.lit(1)).or(Expressions.$("field2").isEqual(Expressions.lit(2L))));
+    Optional<org.apache.iceberg.expressions.Expression> actual = FlinkFilters.convert(expr);
+    Assert.assertTrue(actual.isPresent());
+    Or or = (Or) actual.get();
+    Or expected = (Or) org.apache.iceberg.expressions.Expressions.or(
+        org.apache.iceberg.expressions.Expressions.equal("field1", 1),
+        org.apache.iceberg.expressions.Expressions.equal("field2", 2L));
+
+    Assert.assertEquals(expected.op(), or.op());
+    Assert.assertEquals(expected.left().op(), or.left().op());
+    Assert.assertEquals(expected.right().op(), or.right().op());
+  }
+
+  @Test
+  public void testNot() {
+    Expression expr = resolve(ApiExpressionUtils.unresolvedCall(
+        BuiltInFunctionDefinitions.NOT, Expressions.$("field1").isEqual(Expressions.lit(1))));
+    Optional<org.apache.iceberg.expressions.Expression> actual = FlinkFilters.convert(expr);
+    Assert.assertTrue(actual.isPresent());
+    Not not = (Not) actual.get();
+    Not expected = (Not) org.apache.iceberg.expressions.Expressions.not(
+        org.apache.iceberg.expressions.Expressions.equal("field1", 1));
+
+    Assert.assertEquals(expected.op(), not.op());
+    assertPredicatesMatch((UnboundPredicate<?>) expected.child(), (UnboundPredicate<?>) not.child());
+  }
+
+  @Test
+  public void testLike() {
+    UnboundPredicate<?> expected = org.apache.iceberg.expressions.Expressions.startsWith("field5", "abc");
+    Expression expr = resolve(ApiExpressionUtils.unresolvedCall(
+        BuiltInFunctionDefinitions.LIKE, Expressions.$("field5"), Expressions.lit("abc%")));
+    Optional<org.apache.iceberg.expressions.Expression> actual = FlinkFilters.convert(expr);
+    Assert.assertTrue(actual.isPresent());
+    assertPredicatesMatch(expected, (UnboundPredicate<?>) actual.get());
+  }
+
+  private void matchLiteral(String fieldName, Object flinkLiteral, Object icebergLiteral) {
+    Expression expr = resolve(Expressions.$(fieldName).isEqual(Expressions.lit(flinkLiteral)));
+    Optional<org.apache.iceberg.expressions.Expression> actual = FlinkFilters.convert(expr);
+    Assert.assertTrue(actual.isPresent());
```

Comment:
Assertions should have context so that when one fails it is clear what went wrong. In this case, the assertion is testing that converting succeeded. So this should be `Assert.assertTrue("Conversion should succeed", actual.isPresent())`